### ISSUE TITLE
Sustainability and organization

### ISSUE THREAD

**Issue by DominiqueMakowski** [Sussex-Psychology-Software-Team/DoggoNogo#2]
Now that we are nearing the completion of v0.1, I am wondering what's the best way of organizing things. As mentioned last time, I thought of having 2 repos:

- Sussex-Psychology-Software-Team/DoggoNogoDev: (this one) would be the development one on which we can continue developing in the future when we have some bees. 
- RealityBending/DoggoNogo: the "clean" version used in experiments, would only be updated every new version.

Alternatively, you also mentioned that it was slightly slow to load it from GH, do you think hosting the game on a uni server would make things easier? 

Basically, what do you think about the best way forward in terms of hosting, deployment, ease of updating etc. etc.

**Comment by Max-Lovell** [comment]
Easiest way is to create a fork of a brance for reality bending. Or can clone, copy, and push to another repo.

Might be able to make it quicker from the uni server, but that's probably depending on where it's accessed from. Can ask james for suggestions on that kind of stuff tomorrow.

**Comment by DominiqueMakowski** [comment]
Once we're happy with 0.1 we can push a release here and then I can republish that release in the other repo
